packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -63,6 +63,12 @@
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
+  <PropertyGroup>
+    <devEnabled>true</devEnabled>
```

Comment:
Please consider to add a prefix for the variable to disambiguate with any other MSBuild variables. This name looks too generic to be accidentally used by some other MSBuild task. E.g. it could be ReactNativeDevMode.
Also, it would be nice to use Pascal casing as other property names in this file.

Reply:
I agree something like JSBundleDevMode or something, since its only affecting if the JS bundle. Having said that, the same setting in RN android's gradle builds is just called devEnabled, so there is some precedence to the devEnable name. -- So I'm fine with this too.

Reply:
Actually thats just a local var.  The actual external gradle interface is more complex:
``` 
           // Set up dev mode
            def devEnabled = !(config."devDisabledIn${targetName}"
                || targetName.toLowerCase().contains("release"))```

Reply:
Updated